code_examples/sdk_examples/src/dapp/dapp/01_domain_linkage_ctype.ts

```diff
@@ -2,12 +2,11 @@ import * as Kilt from '@kiltprotocol/sdk-js'
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
 export async function main(): Promise<Kilt.ICType> {
-  const { creator, createdAt, ...domainLinkageCType } =
-    await Kilt.CType.fetchFromChain(
-      'kilt:ctype:0xb08800a574c436831a2b9fce00fd16e9df489b2b3695e88a0895d148eca0311e'
-    )
+  const { creator, createdAt, cType } = await Kilt.CType.fetchFromChain(
```

Comment:
shouldn't we keep the name the same as it is an example?

```suggestion
  const { creator, createdAt, cType: domainLinkageCType } = await Kilt.CType.fetchFromChain(
```

Reply:
why tho????

Reply:
nobody should be using this CType btw, it is wrong and broken

Reply:
definitely not what people should use for the domain linkage credential

Reply:
ok reverted now, guess the issue that this is a shitty ctype is a separate issue which has also wider implications, as it seems to be the one everybody uses.